### PR TITLE
Deployment of Hermes VM binaries to kiwi.com Maven repository.

### PR DESCRIPTION
**Pull request by Blackhex**

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -7,7 +7,7 @@ buildscript {
         mavenLocal()
         google()
         jcenter()
-        reactNativeFromUnpkg(it, reactNativeVersion)
+        packageFromUnpkg(it, "react-native", reactNativeVersion)
```
Comment (tbergq) [inline android/build.gradle]: Why this change? I think react-native is the only package we get from Unpkg

File: android/rnkiwimobile/build.gradle [hunk 2]
```diff
@@ -79,14 +76,19 @@ dependencies {
...
 
-    if (enableHermes) {
-      def hermesPath = "../../node_modules/hermesvm/android/";
-      debugImplementation files(hermesPath + "hermes-debug.aar")
-      stagingReleaseImplementation files(hermesPath + "hermes-release.aar")
-      releaseImplementation files(hermesPath + "hermes-release.aar")
-    } else {
-      implementation jscFlavor
-    }
+    implementation "com.kiwi.android:hermesvm:$hermesVmVersion-SNAPSHOT"
```
Comment (tbergq) [inline android/rnkiwimobile/build.gradle]: This means we will always use the deployed version right?

Could I add this: 

```
def hermesPath = "../../node_modules/hermesvm/android/";	
      debugImplementation files(hermesPath + "hermes-debug.aar")	
      stagingReleaseImplementation files(hermesPath + "hermes-release.aar")
releaseImplementation "com.kiwi.android:hermesvm:$hermesVmVersion-SNAPSHOT"

```

Since we will also use this in the playground app.

File: android/build.gradle (hunk 1 above)
Comment (Blackhex) [reply android/build.gradle]: I tried to use Hermes from Unpkg but the artifact itself have non-Maven structure so it was not unfortunatelly possible. I guess in future there may be some other packages that could be downloaded from Unpkg thus the "generification" of this utility function.

File: android/rnkiwimobile/build.gradle (hunk 2 above)
Comment (Blackhex) [reply android/rnkiwimobile/build.gradle]: Sure, I thought the rnkiwimobile is used only for stagingRelease playground and kiwi.com app.

File: android/build.gradle (hunk 1 above)
Comment (tbergq) [reply android/build.gradle]: Ok, in the end it does not make a big difference to me, I will leave it as is 😊

File: android/rnkiwimobile/build.gradle (hunk 2 above)
Comment (tbergq) [reply android/rnkiwimobile/build.gradle]: There is this snipped in `/playground/build.gradle`:

```
debugImplementation(project(':rnkiwimobile')) {
        exclude group: "com.google.android.gms"
        exclude group: "com.google.firebase"
    }

    // You can choose staging to use the Staging CodePush key
    stagingReleaseImplementation(project(':rnkiwimobile')) {
        exclude group: "com.android.support"
        exclude group: "com.google.android.gms"
        exclude group: "com.google.firebase"
    }

    releaseImplementation("com.trinerdis.skypicker:rnkiwimobile:$rnKiwiMobileVersion-SNAPSHOT") {
        exclude group: "com.android.support"
        exclude group: "com.google.android.gms"
        exclude group: "com.google.firebase"
    }
```

which means it compiles rnkiwimobile right?

**Comment by tbergq** [comment]
@Blackhex, please check the commit I pushed. If it looks good, I will merge this PR.

File: README.md [hunk 3]
```diff
@@ -422,6 +422,6 @@ ios should work just fine. Just do `e2e-iphone` or `e2e-ipad` to run the tests
 
 ## Upgrade react-native
 
-Since we upload our dependencies to maven, and these depends on the current version of react-native, there are no maven packages whenever we upgrade react-native version. In order to test it locally, you need to replace all occurrences of `reactNativeFromUnpkg(it, reactNativeVersion)` with `reactNativeFromNodeModules(it)` and you need to comment `include ':rnkiwimobile'` and `include ':playground'` in settings.gradle. 
+Since we upload our dependencies to maven, and these depends on the current version of react-native, there are no maven packages whenever we upgrade react-native version. In order to test it locally, you need to replace all occurrences of `packageFromUnpkg(it, "react-native", reactNativeVersion)` with `reactNativeFromNodeModules(it)` and you need to comment `include ':rnkiwimobile'` and `include ':playground'` in settings.gradle.
```
Comment (Blackhex) [inline README.md]: with `packageFromNodeModules(it, "react-native")`
